test/integration/tests/toggle_options.sh

```diff
@@ -11,10 +11,22 @@ if [ "$OS" == "FreeBSD" ]; then
     exit 0
 fi
 
-srcdir="$(readlink -e "$(dirname "$0")")"
-toolsdir="$(readlink -e "${srcdir}"/../../../tools)"
-mandir="$(readlink -e "${srcdir}"/../../../man)"
+srcdir="$(readlink -m "$(dirname "$0")")"
```

Comment:
Why would any of these be missing, you're running the test scripts on the host? Can't you just modify TESTS variable to include what you want to run?

Reply:
You're right, in a normally cloned repo, they should be present. Removed this change.